services/email/index.js

```diff
@@ -70,6 +69,15 @@ var caseworkerPlainTextTemplates = {
     path.resolve(__dirname, './templates/caseworker/plain/someone-else.mus')).toString('utf8')
 };
 
+var translationLocation = {
+  error: 'correct-mistakes',
+  'lost-or-stolen-uk': 'lost-stolen-damaged',
+  'lost-or-stolen-abroad': 'lost-stolen-damaged',
+  delivery: 'not-arrived',
+  collection: 'collection',
+  'someone-else': 'someone-else'
+}
```

Comment:
This lot should really not live in here. It puts application logic into a "service"